### DISCUSSION TITLE
Lxqt-powermanagement's battery percentage next to icon.

### DISCUSSION THREAD

**Discussion by ahmadraniri1994** [lxqt/lxqt#2178] [Q&A] [answered]
I tried [this tutorial](https://www.reddit.com/r/Lubuntu/comments/ic1223/lubuntu_2004_battery_percent_indicator/) but it just give me unknown icon, I just need to get battery percentage next to lxqt-powermanagement's icon. Any clue? Other battery applets often just give icon with graphical measurement and not percentage icon next to the applet. Yes, we can hover it and get the battery's percentage, but it will be nice if we can directly know about battery's percentage. Thanks.

**Comment by tsujan** [comment, marked as answer]
See https://github.com/lxqt/lxqt/discussions/2179#discussion-3797615

**Reply by ahmadraniri1994** [reply]
Thanks.

**Reply by tsujan** [reply]
Also, @palinek made a nice icon with the percentage inside it, as an extra option. It'll be in LXQt 1.1.0.

**Comment by palinek** [comment]
FYI ... https://github.com/lxqt/lxqt-powermanagement/pull/294
